tests/integration/users/test_views.py

```diff
@@ -12,7 +12,9 @@ def test_get_anonymous(self):
         response = self.client.get(self.url)
 
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, b'{"user_id":null}')
+        self.assertEqual(
+            json.loads(response.content.decode('utf-8')),
```

Comment:
ok. But you can remove the string 'utf-8'. It is the default value of the argument: https://docs.python.org/3.5/library/stdtypes.html#bytes.decode